docs/querying/sql-data-types.md

```diff
@@ -64,7 +75,7 @@ The following table describes how Druid maps SQL types onto native types when ru
 |TIMESTAMP|LONG|`0`, meaning 1970-01-01 00:00:00 UTC|Druid's `__time` column is reported as TIMESTAMP. Casts between string and timestamp types assume standard SQL formatting, such as `2000-01-02 03:04:05`, not ISO 8601 formatting. For handling other formats, use one of the [time functions](sql-scalar.md#date-and-time-functions).|
 |DATE|LONG|`0`, meaning 1970-01-01|Casting TIMESTAMP to DATE rounds down the timestamp to the nearest day. Casts between string and date types assume standard SQL formatting&mdash;for example, `2000-01-02`. For handling other formats, use one of the [time functions](sql-scalar.md#date-and-time-functions).|
 |ARRAY|ARRAY|`NULL`|Druid native array types work as SQL arrays, and multi-value strings can be converted to arrays. See [Arrays](#arrays) for more information.|
-|OTHER|COMPLEX|none|May represent various Druid column types such as hyperUnique, approxHistogram, etc.|
+|OTHER|COMPLEX|none|May represent various Druid column types such as hyperUnique, cardinality, etc.|
```

Comment:
cardinality isn't a type, this should mention that it is dependent on which extensions are loaded and link to the extension docs. However, afaik none of the extension docs contain information on how the type is displayed in like `INFORMATION_SCHEMA` columns table, so those docs might also need updated to indicate how their types are presented.